sdk/python/feast/infra/online_stores/contrib/postgres.py

```diff
@@ -251,6 +260,49 @@ def teardown(
             logging.exception("Teardown failed")
             raise
 
+    def retrieve_online_documents(
+        self,
+        config: RepoConfig,
+        table: FeatureView,
+        requested_feature: str,
+        embedding: List[float],
+        top_k: int,
+    ) -> List[Tuple[Optional[datetime], Optional[Dict[str, ValueProto]]]]:
```

Comment:
Do we need `Optional[Dict[str, ValueProto]]`? If only a single feature can be searched, wouldn't `Optional[ValueProto]` be sufficient?